src/utils/signMakerOrders.ts

```diff
@@ -15,7 +16,8 @@ export const signMakerOrder = async (
   domain: TypedDataDomain,
   makerOrder: Maker
 ): Promise<string> => {
-  return signer._signTypedData(domain, makerTypes, makerOrder);
+  const signature = await signer._signTypedData(domain, makerTypes, makerOrder);
+  return utils.joinSignature(utils.splitSignature(signature));
```

Comment:
```suggestion
  return utils.joinSignature(signature);
```

`joinSignature` already does `splitSignature` inside, no need to repeat the operation twice;

> https://github.com/ethers-io/ethers.js/blob/master/packages/bytes/src.ts/index.ts#L474